status-im/status-react
# Allow user to recover unencrypted account

fixes: #4883
fixes: #4800 
fixes: #4126 (This issue is due to the fact that when we added encryption we did not handle resetting realm correctly, only deleting `default.realm` but none of the realm accounts)

Allow the use to keep using an invalid key in order to recover their account data.

### Testing notes

I have changed the copy to `Cancel` as we don't allow overriding that bit of text, and probably best to keep the changes to a minimum.
Title has also been changed in order to fit.

1) Uninstall any previous version
2) Install this version of the app
apk uploaded to https://i.diawi.com/XaiwK2
ipa uploaded to https://i.diawi.com/kxJT6V

3) Once you see the popup click on `Cancel`, create an account
4) You are now in the error condition, database is unencrypted
5) Install this version of the app
apk uploaded to https://i.diawi.com/QPS4Dy
ipa uploaded to https://i.diawi.com/1pGTik
6) On startup you should see the pop up. Clicking on `Cancel` should allow you to continue as usual. You will see the popup every time you start the app/logout, clicking on `Apply` will reset the database and you should not see the pop up again.

status: ready
